index.js

```diff
@@ -0,0 +1,66 @@
+// Task 1: Even or Odd Checker
+const number = 7;
```

Comment:
აქ `prompt` უნდა გამოგეყენებინა
```suggestion
const number = Number(prompt("Enter a number"));
```